src/generic/configure-modal/messages.js

```diff
@@ -239,6 +239,10 @@ const messages = defineMessages({
     id: 'course-authoring.course-outline.configure-modal.advanced-tab.time-allotted',
     defaultMessage: 'Time allotted (HH:MM):',
   },
+  timeLimitPlaceholder: {
+    id: 'course-authoring.course-outline.configure-modal.advanced-tab.time-limit-placeholder',
+    defaultMessage: 'HH:MM',
```

Comment:
Please add a `description: ` for this message. In particular, since the form field has `pattern="^[0-9][0-9]:[0-5][0-9]$"`, the translator MUST use the format "XX:YY" exactly